pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/ProxyConnection.java

```diff
@@ -376,6 +395,14 @@ private synchronized void completeConnect() throws PulsarClientException {
 
             state = State.ProxyLookupRequests;
             lookupProxyHandler = service.newLookupProxyHandler(this);
+            if (service.getConfiguration().isAuthenticationEnabled()
+                    && service.getConfiguration().getAuthenticationRefreshCheckSeconds() > 0) {
+                authRefreshTask = ctx.executor().scheduleAtFixedRate(
+                        Runnables.catchingAndLoggingThrowables(() -> refreshAuthenticationCredentials(false)),
```

Comment:
It will never refresh the authentication credential.

`(!authState.isExpired() || !force)` always `true`.

Reply:
Great catch, that is a bad mistake. I'll fix it now.